src/ImpactX.cpp

```diff
@@ -102,6 +102,5 @@ namespace impactx
         diagnostics::DiagnosticOutput(*m_particle_container,
                                       diagnostics::OutputType::PrintParticles,
                                       "diags/output_beam.txt");
```

Comment:
Removing unrelated change
```suggestion
                                      "diags/output_beam.txt");

```